metaflow/plugins/cards/card_cli.py

```diff
@@ -0,0 +1,388 @@
+from metaflow.client import Task
+from metaflow import JSONType, namespace
+from metaflow.exception import CommandException
+import webbrowser
+import re
+import click
+import os
+import signal
+import random
+from contextlib import contextmanager
+from functools import wraps
+from metaflow.exception import MetaflowNamespaceMismatch
+from .card_datastore import CardDatastore, NUM_SHORT_HASH_CHARS
+from .exception import (
+    CardClassFoundException,
+    IncorrectCardArgsException,
+    UnrenderableCardException,
+    CardNotPresentException,
+)
+
+from .card_resolver import resolve_paths_from_task, resumed_info
+
+
+def serialize_flowgraph(flowgraph):
```

Comment:
Add TODO to move to the artifact; I think that was the idea (but it hasn't been reviewed yet).

Reply:
Added FIXME tag